packages/formation-react/src/components/ErrorableDate/ErrorableDate.stories.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import ErrorableDate from './ErrorableDate';
+
+export default {
+  title: 'Library/ErrorableDate',
+  component: ErrorableDate,
+  argTypes: {
+    required: { control: 'boolean' },
+    label: { control: 'text' },
+    name: { control: 'text' },
+    requiredMessage: { control: 'text' },
+    invalidMessage: { control: 'text' },
+  },
+};
+
+const Template = (args) => {
+  const [date, setDate] = useState({
```

Comment:
Heh, I had to do the same thing with the `AdditionalInfo` component. It took me _entirely_ too long to figure out the right solution—maybe because it's been so long since I've used React. Would you have found it helpful if we had some short documentation on how to wrap the component in a HOC that handles the state for these stories?

Reply:
I agree I was stumped for a sec too :laughing: ... any docs surrounding our process is always helpful where would something like that live?

Reply:
Honestly, we can throw a `.mdx` file in here and the documentation for how to create interactive stories can be right here in Storybook. :smile:

Reply:
oh hahahaha I thought you leaning towards some other docs ill update it tks! :rofl: